### ISSUE TITLE
Using custom webpack loaders

### ISSUE THREAD

**Issue by akeelnazir** [angular/angular-cli#8106]
### Bug Report or Feature Request (mark with an `x`)
```
- [X ] feature request
```

### Versions.
1.4.3

### Desired functionality.
I have to load a custom loader to process .graphql files, I'd like to be able to add custom loaders to webpack config.

### Mention any other details that might be useful.
The rule I want to add is
```
/* File loader for graphql files.
*  Returns file content as string
*/
  {
     test: /\.(graphql|gql)$/,
     exclude: /node_modules/,
     loader: 'graphql-tag/loader'
  }
```

**Comment by filipesilva** [comment]
I know this is not the answer you wanted but truth is this has been extensively discussed before, and so this issue is a duplicate of https://github.com/angular/angular-cli/issues/1656. You can read there what the final word on that topic is for now.

**Comment by JohannesHoppe** [comment]
@akeelnazir 

I came around here, because I have the exact same use-case.  I want to use the `graphql-tag/loader` but without the burden of `ng eject`. You can do this instead:

```js
const myQuery = require('graphql-tag/loader!./myQuery.graphql');
```

**Comment by plck** [comment]
@JohannesHoppe 
How and where exactly would you use this 🤔 ? Could you maybe give more explanation? Thanks!

**Comment by JohannesHoppe** [comment]
The original poster obviously wants to import *.graphql files, for apollo. The usual way would be a configured webpack-loader. This is not possible with the angular-cli (exception: you `ng eject` the application). My require-statement does the same job and requires no changes to the webpack configuration. Cheers!

**Comment by FloNeu** [comment]
Hello! Thx, guys - needed to do this too - the require works fine, but wondering... is there currently a better supported way of doing this... i would very much like to just use import-statement.

**Comment by camsong** [comment]
Blocked by this issue, any updates in 2019???
@filipesilva

**Comment by camsong** [comment]
@hansl please be open, this made many developers drop Angular to choose React and Vue!!!
